Makefile

```diff
@@ -161,21 +162,21 @@ verify: verify-edit verify-merged
 
 # Run validation queries on obi-edit and exit on error.
 .PHONY: verify-edit
-verify-edit: src/ontology/obi-edit.owl  $(EDIT_VIOLATION_QUERIES) | build
+verify-edit: src/ontology/obi-edit.owl $(EDIT_VIOLATION_QUERIES)
 	$(ROBOT) verify --input $< --output-dir build \
 	--queries $(EDIT_VIOLATION_QUERIES)
 
 # Run validation queries on obi_merged and exit on error.
 .PHONY: verify-merged
-verify-merged: build/obi_merged.owl $(MERGED_VIOLATION_QUERIES) | build
+verify-merged: build/obi_merged.owl $(MERGED_VIOLATION_QUERIES)
```

Comment:
Should be `verify-merged: build/obi_merged.owl $(MERGED_VIOLATION_QUERIES) | build/robot.jar`